### DISCUSSION TITLE
Shall we ever "Close" discussions ?

### DISCUSSION THREAD

**Discussion by smerkel** [dmi-3D/dmi-3D.github.io#38] [General]
Hi all,

I realized that, when we close a discussion, the discussion does not show up in the discussion list anymore. One has to select **_Filter:all_**  to see resolved discussions.

New users will never think of checking this option when connecting to the discussion board at first. 

Hence, my question: shall we ever close a discussion?

Sébastien

**Comment by jonwright** [comment]
Oups. https://github.com/orgs/community/discussions/64980
Perhaps we should re-open any that are 'closed'?

**Comment by smerkel** [comment]
I believe so, but I wanted to get other people's opinion. I think it is best to keep the discussions open. It will be easier for new users to find content.
